### PR TITLE
pt-br translation for Transact Safely, Run a Full Node

### PR DESCRIPTION
**Pull request by rating89us**

### CODE REVIEW

**Comment by erciccione** [comment]
Thanks @rating89us :)

pinging @netrik182 @vp1111 @botoserelepe for reviews

**Comment by netrik182** [comment]
I'm on it right now.

File: monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Faça Transações com Segurança, Use um Nodo Completo
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md]:
```suggestion
# Faça Transações com Segurança, Rode um Nó Completo
```
I would like to ask that you change all translations of "nodo" to "nó" as it is already widely used across the Monero Ecosystem ([Monerujo](https://github.com/m2049r/xmrwallet/search?q=n%C3%B3+remoto&unscoped_q=n%C3%B3+remoto), [Official Website](https://ww.getmonero.org/pt-br/resources/moneropedia/), [Official GUI Wallet](https://github.com/monero-project/monero-gui/search?q=n%C3%B3&unscoped_q=n%C3%B3) and so on.)

File: monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Ao usar a carteira Monero GUI (com interface gráfica), o nodo completo também é executado pelo daemon. Ele pode ser executado automaticamente durante a abertura da carteira, melhorando a experiência do usuário através da integração da carteira com o processo de sincronização da blockchain. O site Monero.how possui um guia fácil para vários sistemas.
+
+**Minerando**: é necessário ter um nodo completo para se minerar. Com um nodo completo do Monero você pode minerar sozinho, contribuindo para manter a segurança da rede (Veja o artigo: [**Minere para Apoiar a Rede**](https://www.monerooutreach.org/mine-to-support-the-network.php)). Rodar um nodo completo ajuda os mineradores e outros nodos a saberem que eles estão na cadeia correta, fortalecendo a blockchain e dificultando a realização de ataques. De fato, os nodos do Monero só precisam estar conectados a um nodo honesto para terem a garantia de que eles estão na cadeia correta.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md]:
```suggestion
**Minerando**: é necessário ter um nodo completo para se minerar. Com um nodo completo do Monero você pode minerar sozinho, contribuindo para manter a segurança da rede (Veja o artigo: [**Minere para Apoiar a Rede**](https://www.monerooutreach.org/praticas_recomendadas_monero/mina-para-apoiar-a-rede.php)). Rodar um nodo completo ajuda os mineradores e outros nodos a saberem que eles estão na cadeia correta, fortalecendo a blockchain e dificultando a realização de ataques. De fato, os nodos do Monero só precisam estar conectados a um nodo honesto para terem a garantia de que eles estão na cadeia correta.
```
I think we could link to the Portuguese article. Also pinging @lh1008 here.

File: monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+**Nodos Remotos**: os nodos remotos são nodos completos administrados por terceiros que fornecem um serviço de compartilhamento de nodos, permitindo que outros usuários se conectem a eles. Os usuários podem se conectar aos nodos remotos para fazerem transações com Monero imediatamente, sem precisarem esperar pela sincronização de um nodo completo. Ao longo dos anos, diversas melhorias na utilização dos nodos remotos foram implementadas pelos desenvolvedores do Monero.
+
+Embora o serviço dos nodos remotos seja muito útil, aumentando a usabilidade e a adoção da moeda, ele também pode afetar um dos pilares essenciais da privacidade da rede Monero. Os nodos remotos podem registrar o endereço IP, a "altura de bloco" e outros dados sobre blocos dos clientes que se conectam a eles. Se você se conecta à rede Monero apenas através de nodos remotos de terceiros, você deve sempre ter essas limitações em mente.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md]:
```suggestion
Embora o serviço dos nodos remotos seja muito útil, aumentando a usabilidade e a adoção da moeda, ele também pode afetar um dos pilares essenciais da privacidade da rede Monero. Os nodos remotos podem registrar o endereço IP, a "altura de restauração" e outros dados sobre blocos dos clientes que se conectam a eles. Se você se conecta à rede Monero apenas através de nodos remotos de terceiros, você deve sempre ter essas limitações em mente.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Ao usar um nodo remoto, você está confiando em outras pessoas para verificar e transmitir as suas transações para a rede, delegando para elas a escolha das regras de consenso da rede. Alguns dos operadores de nodos remotos conhecidos pela comunidade incluem a [**monerujo**](https://www.monerujo.io/) (carteira light para android), [**cakewallet**](https://cakewallet.io/) (carteira light para iOS) e [**mymonero**](https://mymonero.com/) (carteira light para web/desktop).
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md]:
```suggestion
Ao usar um nodo remoto, você está confiando em outras pessoas para verificar e transmitir as suas transações, delegando para elas a escolha das regras de consenso da rede. Alguns dos operadores de nodos remotos conhecidos pela comunidade incluem, porém não se limitam a: moneroworld, [**monerujo**](https://www.monerujo.io/) (carteira light para android), [**cakewallet**](https://cakewallet.io/) (carteira light para iOS) e [**mymonero**](https://mymonero.com/) (carteira light para web/desktop).
```

File: monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+**Conclusão**
+Agora que você está melhor informado, mantenha-se vigilante e busque usar todos os meios de privacidade disponíveis que você tiver! Para aprender mais sobre as melhores maneiras de se manter seguro enquanto usa Monero, visite: monerooutreach.org/best-practices-for-monero.php
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/PSA/files/run_node_pt-br.md]:
```suggestion
Agora que você está melhor informado, mantenha-se vigilante e busque usar todos os meios de privacidade disponíveis que você tiver! Para aprender mais sobre as melhores maneiras de se manter seguro enquanto usa Monero, visite: https://www.monerooutreach.org/praticas_recomendadas_monero/
```
Also use here the Portuguese version of monerooutreach.org

**Comment by rating89us** [comment]
Updated

**Comment by erciccione** [comment]
For future reference: keep in mind that we have a [Monero terminology guide](https://github.com/monero-ecosystem/monero-translations/blob/master/terminology-guides/portuguese-br-terminology.md) for pt-br. All translations should be consistent with terminology there. If the guide is not up to date, it should be updated.

**Pull request merged**